hist/hist/v7/test/atomics.cxx

```diff
@@ -29,20 +26,35 @@ std::atomic<FLOAT> &atomic_addeq(std::atomic<FLOAT> &lhs, FLOAT rhs,
    return lhs;
 }
 
-// THistImpl uses op+= to add to bin content; provide the relevant overload.
-template <class PRECISION>
-std::atomic<PRECISION> &operator+=(std::atomic<PRECISION> &lhs, PRECISION rhs)
-{
-   return atomic_addeq(lhs, rhs);
-}
+template <class T>
+struct WrappedAtomic {
```

Comment:
I can't remember the reason why std::atomic is not copyable but we should enhance this class' documentation to make sure it is not mis-used (i.e. inadvertently used in a way that the standard was afraid std::atomic would be used).

Reply:
PS. Even-though it is 'just' a test, it could be copy/pasted by 'clever' users ...

Reply:
It's mainly because you cannot expect this to be an atomic operation. Yeah whatever. If you fill while copying you get what you get - which is fine in this case. I'll document that.